src/store/mod.rs

```diff
@@ -168,7 +168,7 @@ impl StoreConfig {
         match branches {
             2 => std::cmp::min(max_rows_to_discard, 7),
             4 => std::cmp::min(max_rows_to_discard, 5),
-            _ => std::cmp::min(max_rows_to_discard, 3),
+            _ => std::cmp::min(max_rows_to_discard, 2),
```

Comment:
It's not obvious to me why this change is desired or correct. Can you explain? I probably am just not visualizing the flow of when this can be called or how it is used. Let's consider the case where a user has set a default `rows_to_discard` of 4 (just for example). Will this expression evaluating to 2 ever cause any problems? I guess the point is that this function won't be called at all in that case. But what motivates a change in the default here and now?

Reply:
This is a method that provides a default, so setting a default (or having a pre-set default) is not related at this point.  `rust-fil-proofs` (as of PR https://github.com/filecoin-project/rust-fil-proofs/pull/1220) no longer uses this method at all, however if it (i.e. current master), the 'default' value provided does not match the default that `rust-fil-proofs` defaults to, which is the main issue.  Since it's no longer used, this could be not updated, but since it bit us once ...

Reply:
Got, thank you.